independent-projects/resteasy-reactive/common/runtime/src/main/java/org/jboss/resteasy/reactive/common/core/AbstractResteasyReactiveContext.java

```diff
@@ -222,9 +226,6 @@ protected void requestScopeDeactivated() {
      * Ensures the CDI request scope is running when inside a handler chain
      */
     public void requireCDIRequestScope() {
-        if (!running) {
-            throw new RuntimeException("Cannot be called when outside a handler chain");
-        }
```

Comment:
This check doesn't really make sense because calling `requireCDIRequestScope` outside a handler will be problematic in other ways